deployments/docker/src/certs/vts.key

```diff
@@ -0,0 +1,8 @@
+-----BEGIN EC PARAMETERS-----
+BggqhkjOPQMBBw==
+-----END EC PARAMETERS-----
```

Comment:
these aren't redundant because the key already encodes the named curve (in this case prime256v1):
```
    0:d=0  hl=2 l= 119 cons: SEQUENCE
    2:d=1  hl=2 l=   1 prim: INTEGER           :01
    5:d=1  hl=2 l=  32 prim: OCTET STRING      [HEX DUMP]:6251F0C81FB3113CAC43082426D8D81255E4A65863F8FA41BFBBE1AF9C8F7BCD
   39:d=1  hl=2 l=  10 cons: cont [ 0 ]
   41:d=2  hl=2 l=   8 prim: OBJECT            :prime256v1
   51:d=1  hl=2 l=  68 cons: cont [ 1 ]
   53:d=2  hl=2 l=  66 prim: BIT STRING
```

Reply:
??? I feel like I'm missing context here.

Reply:
see https://github.com/veraison/services/pull/234/files#r1643093299